.eslintrc.js

```diff
@@ -143,6 +142,11 @@ module.exports = {
       files: ['*.config.[tj]s', 'packages/website/pages/**/*.tsx', '*.stories.tsx'],
       rules: { 'import/no-default-export': 0 },
     },
+    {
+      // Enable eslint-plugin-testing-library rules only for test files
+      files: ['**/__tests__/**/*.[jt]s?(x)', '**/?(*.)+(spec|test).[jt]s?(x)'],
```

Comment:
That's a great idea to add this regex and use some plugins only for test files. I think we can make it a little better if we:
1. Define files only as `'**/?(*.)+(spec|test).ts?(x)'` as we don't have __tests__ directories and tests written as .js/.jsx files.
2. Move `'plugin:jest/recommended'` and `'plugin:jest-dom/recommended'` here as well.